### PR TITLE
updated-gmailbigquery

### PR DESCRIPTION
**Pull request by JainM6**
## Purpose of this pull request

Made some fixes in the new branch

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/gmail-bigquery.md [hunk 1]
```diff
@@ -49,17 +42,17 @@ Metadata fields will be set, if the integration is configured with the SIEM forw
...
+* `name`
+* `projectId`
+* `datasetId`
+* `privateKey`
+* `clientEmail`
+* `tokenURI`
+* `dataLocation`
+
+:::info
+* User can find `projectId`, `privateKey`, `clientEmail` and `tokenURI` from the Google Cloud Console
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/gmail-bigquery.md:54]: Our style is to use "you", not "user"
```suggestion
* You can find `projectId`, `privateKey`, `clientEmail` and `tokenURI` from the Google Cloud Console
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/gmail-bigquery.md [hunk 2]
```diff
@@ -49,17 +42,17 @@ Metadata fields will be set, if the integration is configured with the SIEM forw
...
+* BigQuery DataSetId  = `datasetId`
+* BigQuery DataSet Location  = `dataLocation`
 * Default value of startTime is "24 Hours ago", user can change it later
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/gmail-bigquery.md:57]:
```suggestion
* You can change the `startTime`, if desired (default value is "24 Hours ago")
```

**Review by kimsauce — DISMISSED** [review]
style edits

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/gmail-bigquery.md (hunk 2 above)
Comment (JainM6) [reply docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/gmail-bigquery.md:57]: Thanks for the suggestions. As these are not complete clauses so in order to maintain consistency. We can decide what goes with the doc. I prefer to keep it as is.

Comment (kimsauce) [reply docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/gmail-bigquery.md:57]: Regardless, it's still our style to use "you", not "user". Please update those.

cc @stacykor

**Review by stacykor — APPROVED** [review]
LGTM

**Pull request merged**